doc/news/changes/major/20240718Thiele

```diff
@@ -0,0 +1,3 @@
+New: LinearAlgebra::TpetraWrappers preconditioner interfaces for Trilinos Ifpack2.
```

Comment:
Given the remark in https://github.com/dealii/dealii/pull/17293#pullrequestreview-2187254861, would you mind listing all new classes that were introduced here? I think that would make the overview more comprehensive.

Reply:
Done.